teuthology/task/install.py

```diff
@@ -577,6 +529,23 @@ def remove_sources(ctx, config):
             p.spawn(remove_fn, remote, project)
 
 
+def get_package_list(config):
+    debug = config.get('debuginfo', False)
+    project = config.get('project', 'ceph')
+    yaml_path = os.path.join(os.path.dirname(__file__), 'packages.yaml')
```

Comment:
can we have 2 files then packages-default which can stay here, and packages-qa.yaml in qa-suites so that  install task yaml will look nicer without big overrides and just use the packages-qa.yaml if it exists?